src/features/landing/components/AuthSection/index.tsx

```diff
@@ -23,9 +30,45 @@ function AuthSection() {
     formState: { errors },
   } = useForm<RegisterItem>();
   const onSubmit = (data: RegisterItem) => {
-
     console.log(data);
   };
+
+  const isLengthValid = (value: string) =>
+    value.length >= 8 && value.length <= 64;
+
+  const hasNumber = (value: string) => /[0-9]/.test(value);
+
+  const hasUppercase = (value: string) => /[A-Z]/.test(value);
+
+  function validatePassword(value: string) {
```

Comment:
write functions in one format or in arrow declaration or function declaration, we need consistency in code

Reply:
Done